src/box/iproto.cc

```diff
@@ -2961,15 +2961,14 @@ iproto_do_cfg(struct iproto_thread *iproto_thread, struct iproto_cfg_msg *msg)
 	return 0;
 }
 
-static inline int
+static inline void
 iproto_send_stop_msg(void)
 {
 	struct iproto_cfg_msg cfg_msg;
 	iproto_cfg_msg_create(&cfg_msg, IPROTO_CFG_STOP);
 	for (int i = 0; i < iproto_threads_count; i++)
-		if (iproto_do_cfg(&iproto_threads[i], &cfg_msg) != 0)
-			return -1;
-	return 0;
+		if (iproto_do_cfg(&iproto_threads[i], &cfg_msg))
+			panic("failed to stop listen");
```

Comment:
AFAIU this can't possibly fail so it should be an assertion. Also, this looks like an independent change that should be done in a separate patch.

Same comment for other similar changes.

Reply:
fixed